Dockerfile

```diff
@@ -1,5 +1,6 @@
 # MTConnect Public C++ Agent Docker image build instructions
 
+# TODO: Do we need notes like this to be in `Dockerfile`? I think the docs might be a better place for this. IMHO most notes in this file should be moved to the docs.
```

Comment:
I wanted to remove these `TODO` comments before merging PR, but after discussing them further.

@wsobel, what do you think about moving these `notes` to the docs? :thinking: